lib/sles4sap.pm

```diff
@@ -135,6 +136,49 @@ undef by default. Test modules testing for SAP ASE should set this property befo
 
 has ASE_RESPONSE_FILE => undef;
 
+=head2 download_hana_assets_from_server
+
+    $self->download_hana_assets_from_server()
+
+Download and extract HANA installation media to /sapinst directory of the SUT.
+The media location must be provided as ASSET_0 in the job settings and be
+available as an uncompressed tar in the factory/other directory of the openQA
+server
+
+=cut
+
+sub download_hana_assets_from_server {
+    my %params = @_;
+    my $target = $params{target} // '/sapinst';
+    my $nettout = $params{nettout} // 2700;
+    script_run "mkdir $target";
+    assert_script_run "cd $target";
+    my $filename = get_required_var('ASSET_0');
```

Comment:
Move line 156 before 154. So if the job is triggered without the proper variables, the mkdir is not executed too.

Reply:
done